julia/core.py

```diff
@@ -141,6 +142,10 @@ class JuliaImporter(object):
     # find_module was deprecated in v3.4
     def find_module(self, fullname, path=None):
         if fullname.startswith("julia."):
+            pypath = os.path.join(os.path.dirname(__file__),
+                                  "{}.py".format(fullname[len("julia."):]))
+            if os.path.isfile(pypath):
```

Comment:
We need to fallback to standard Python importer in this case to import Python submodules (e.g., `julia/magic.py`).

Reply:
Of course, this has to be improved if/when we have python subpackages.